packages/cardhost/app/styles/index.css

```diff
@@ -30,6 +30,14 @@
   background: url(/assets/images/cardstack-logo.svg) 30px 30px/24px 25px no-repeat;
 }
 
+.cardhost-welcome-left-edge--nav button {
```

Comment:
@jenweber Since `button`s bring their own font properties and the whole family to the party, I'd add the below. Everything else looks good.
```
    font: 400 13px/18px var(--ch-font-family);
    letter-spacing: 0.015em;
```

Reply:
Added, thank you!